### DISCUSSION TITLE
In there instruction how to write own script for creation of LXC?

### DISCUSSION THREAD

**Discussion by beautyfree** [tteck/Proxmox#1616] [Help Needed] [answered]
I would like to write my own script, because unfortunately the implementation of ideas does not happen so often

Also I ran into some problem, I can’t enter the newly created LXC - it asks for a login and password, although this is not the case with other LXCs from tteck (and it's not clear how to debug steps when creating a container)

and against this background, a request arose for an instruction that would possibly attract more developers to contribute to this repository

**Comment by tteck** [comment, marked as answer]
By reading and comprehending https://github.com/tteck/Proxmox/blob/main/CODE-AUDIT.md, the process should become straightforward.
